File: 12th March, 2023/Binary matrix having maximum number of 1s.py
#User function Template for python3

class Solution:
    def findMaxRow(self, mat, N):
        # Code here
        row=0; j=N-1; res = [0]*2;
        for i in range(N):
            while(j>=0 and mat[i][j]==1):
                row=i
                j-=1
        res[0] = row
        res[1] = N-j-1
        return res

File: 12th March, 2023/Binary matrix having maximum number of 1s.java


//User function Template for Java

class Solution {
    public int[] findMaxRow(int mat[][], int N) {
        // code here
        int row=0; int j=N-1; int res[] = new int[2];
        for(int i=0; i<N; i++)
        {
            while(j>=0 && mat[i][j]==1)
            {
                row=i;
                j--;
            }
        }
        res[0] = row;
        res[1] = N-j-1;
        return res;
    }
};
